Cannot find module './package.json'

node: v6.11.1
package version: 1.1.2

i am receiving the next message when i try to generate a backup Cannot find module './package.json'

module.js:471
    throw err;
    ^

Error: Cannot find module './package.json'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\usuario\AppData\Roaming\npm\node_modules\firestore-backup-restore\build\index.js:47:15)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)

--- a/index.js
+++ b/index.js
@@ -21,22 +21,26 @@ const accountCredentialsPathParamDescription =
   'Google Cloud account credentials JSON file';
 
 const backupPathParamKey = 'backupPath';
-const backupPathParamDescription = 'Path to store backup.';
+const backupPathParamDescription = 'Path to store backup';
 
 const restoreAccountCredentialsPathParamKey = 'restoreAccountCredentials';
 const restoreAccountCredentialsPathParamDescription =
-  'Google Cloud account credentials JSON file for restoring documents.';
+  'Google Cloud account credentials JSON file for restoring documents';
 
 const prettyPrintParamKey = 'prettyPrint';
-const prettyPrintParamDescription = 'JSON backups done with pretty-printing.';
+const prettyPrintParamDescription = 'JSON backups done with pretty-printing';
 
 const plainJSONBackupParamKey = 'plainJSONBackup';
-const plainJSONBackupParamDescription = `JSON backups done without preserving any type information.
-                                          - Lacks full fidelity restore to Firestore.
-                                          - Can be used for other export purposes.`;
+const plainJSONBackupParamDescription = `JSON backups done without preserving any type information
+                                          - Lacks full fidelity restore to Firestore
+                                          - Can be used for other export purposes`;
 
 const packagePath = __dirname.includes('/build') ? '..' : '.';
-const version = require(`${packagePath}/package.json`).version;
+
+let version = 'N/A - unable to read package.json file';
+try {
+  version = require(`${packagePath}/package.json`).version;
+} catch (requireError) {}
 
 // The data to be restored can replace the existing ones
 // or they can be merged with existing ones

